### ISSUE TITLE
Multifile editor - visual indication which editor is for which file

### ISSUE THREAD

**Issue by gikf** [freeCodeCamp/freeCodeCamp#54713]
### Is your feature request related to a problem? Please describe.

Multifile editor, as the name suggests, has multiple editors (tabs), each for different file. Each editor can be hidden or shown by their own button, however there isn't really any visual (or text) indication which editor corresponds to which button and therefore to which file.

To be clear, this doesn't seem to create much confusion, if any. I don't recall anybody having issue with mixing up the tabs/editors, both buttons and editors always follow the same ordering.

Nevertheless I think it'd be nice to have a more visible connection between individual buttons and their corresponding editors.

### Describe the solution you'd like

The easiest approach might be just including comment at the top of file, similarly to how it's added when creating post on forums via the help button. That doesn't sound ideal though.

Not entirely sure if monaco perhaps has some feature which might be used for this.

### Describe alternatives you've considered

Slightly different way to achieve this, might be introducing some additional color for button when corresponding editor is focused.

**Comment by a2937** [comment]
I'm actually a pretty huge fan on the additional color when that editor is in focus. But the colors have to distinctive and mesh well with our themes. We do have an purple, a yellow, a blue, and a green we can easily use already defined as accent colors in our style guide. I'm leaning towards purple for html, blue for CSS, and green for Javascript.

**Comment by bbsmooth** [comment]
I just want to make sure I understand the color suggestion. If I am understanding correctly, when keyboard focus is placed in one of the editors, its corresponding button would get a different color than the other buttons to make it more obvious which editor has focus? The accessibility issue here is that this new color must have a `3:1` contrast with both the black button and the light gray button, since you could have all three colors showing at one time.

**Comment by gikf** [comment]
Yeah, when focused is editor corresponding to specific button, change it color, border or 🤷‍♂️.

**Comment by naomi-lgbt** [comment]
Would this not also fall under "don't use colour as the only visual indicator that something's different"

**Comment by bbsmooth** [comment]
As long so the color difference has a contrast ratio of at least `3:1` then it passes WCAG. If that is not the case then we would most likely need to add something else as an indicator, but it could be as simple as a little dot next to the name of the active file. Maybe we could get some mock-ups to get a clearer picture of how this would work?

**Comment by ahmaxed** [comment]
how about just keeping things simple.

<img width="929" alt="Image" src="https://github.com/user-attachments/assets/fb3cfe05-125f-408a-8090-79f27a6dae24" />

**Comment by a2937** [comment]
That's perfect.

**Comment by ahmaxed** [comment]
For the implementation, the file name should not be scrollable, should have the same height as an editor line, the name should be aligned with the editor numbers, and the text should be in bold.

**Comment by c0d1ng-ma5ter** [comment]
Just a random question, which files should be changed?

**Comment by gikf** [comment]
> Just a random question, which files should be changed?

@c0d1ng-ma5ter, likely at least [client/src/templates/Challenges/classic/editor.tsx](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/templates/Challenges/classic/editor.tsx), possibly also [client/src/templates/Challenges/classic/multifile-editor.tsx](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/templates/Challenges/classic/multifile-editor.tsx).
